### DISCUSSION TITLE
Is "Crowdstrike Falcon for VMWare Tanzu" v1.1.0 compatible with Jammy stemcell

### DISCUSSION THREAD

**Discussion by David-VMware** [CrowdStrike/falcon-boshrelease#8] [Q&A]
Hi Team,
Thanks for reviewing this issue. Is "Crowdstrike Falcon for VMWare Tanzu" v1.1.0 compatible with Jammy stemcell?

After reviewing the files inside crowdstrike-falcon-cf-1.1.0.pivotal, I can see it is compatible with Xenial only. However, [Xenial reaches end of support in April 2024](https://docs.vmware.com/en/Stemcells-for-VMware-Tanzu/services/release-notes/index.html#xenial-stemcells-1), which brings severe challenges related to compatibility and deployment on expired product.
`stemcell_criteria:`
`  os: ubuntu-xenial`
`  requires_cpi: false`
`  version: '621'`

If it is compatible with Jammy stemcell, when can we have the next release available?

**Comment by David-VMware** [comment]
By the way, I can see it can run in VMs deployed with Jammy stemcell.
`- name: falcon-linux-sensor-config`
`  runtime_config: |`
`      addons:`
`      - include:`
`          stemcell:`
`          - os: ubuntu-jammy`
`          - os: ubuntu-bionic`
`          - os: ubuntu-xenial`
`          - os: ubuntu-trusty`

**Comment by redhatrises** [comment]
@David-VMware you will notice that the tile.yml does not specify the stemcell_criteria because there are no errands or deployments used in this tile, which if I understand correctly, means setting the stemcell_criteria is unnecessary since it is unused. Since this tile is only an addon at the moment, all the Ubuntu versions we support are listed in the addon's inclusion rules which will deploy the addon to all the deployments that meet the inclusion rule criteria.

**Comment by David-VMware** [comment]
Hi @redhatrises ,
Thanks for your detailed explanation. I understand stemcell_criteria is unnecessary. The reason why I request changing to `ubuntu-jammy` from `ubuntu-xenial` is that the ubuntu-xenial stemcell must exist in the Tanzu ops manager. Or the deployment will fail because a preliminary check finds the requested stemcell missing even though we know it is not used. This causes extra burden to maintain obsolete and out-of-support ubuntu-xenial stemcell from the end users. 
![Screenshot 2024-09-24 at 09 53 33](https://github.com/user-attachments/assets/e651c3ba-18da-4e9c-99fc-1071532a682d)

**Comment by redhatrises** [comment]
@David-VMware we actually currently don't use or deploy any Stemcells, so configuring CrowdStrike in the Stemcell Library currently does nothing for the CrowdStrike tile and deployment. Since there is no stemcell used, a stemcell does not have to be configured or exist for CrowdStrike to deploy, AND currently, nothing will show in the deployed column either since once again no stemcell is used in deployment. We have several customers who have deployed without touching the Stemcell library. Here's an example of a deployment that was just done today:

1. Unconfigured Stemcell since it is not needed or used
   
    ![slib](https://github.com/user-attachments/assets/e569255d-c139-40dc-bf81-86f754bd48ea)

2. Selected configuration for bosh to deploy the addon to the Tanzu infrastructure environment

    ![deployment](https://github.com/user-attachments/assets/00a4b975-22e9-4ea5-a202-fc12de769ac8)

3. Changelog post deployment showing the CrowdStrike Falcon addon was deployed successfully without a Stemcell

    ![changlog](https://github.com/user-attachments/assets/d6544978-77b3-4044-905e-8afba42040bc)

4. CrowdStrike Falcon tile status shows no resources and is empty since no stemcell is used

    ![resources](https://github.com/user-attachments/assets/2a6091b4-07ce-487f-bd08-9e4e64045630)
